packages/babel-plugin/src/utils/css-map.ts

```diff
@@ -17,8 +37,12 @@ export const getKeyValue = (key: ObjectKeyWithLiteralValue): string => {
   throw new Error(`Expected an identifier or a string literal, got type ${(key as any).type}`);
 };
 
-export const isAtRule = (key: ObjectKeyWithLiteralValue): key is ObjectKeyWithLiteralValue =>
-  getKeyValue(key).startsWith('@');
+export const isAtRuleObject = (
+  key: ObjectKeyWithLiteralValue
+): key is ObjectKeyWithLiteralValue => {
+  const keyValue = getKeyValue(key);
```

Comment:
So this value must exactly match as this is always explicitly like `cssMap({ primary: { '@media': { '(min-width: 400rem)': { … } } } });` right?  LGTM…

Reply:
Yes exactly.